pkgs/games/runelite/default.nix

```diff
@@ -1,19 +1,32 @@
-{ stdenv, fetchurl, makeWrapper, jre, makeDesktopItem, lib }:
+{ stdenv, lib, pkgs, ... }:
+
+with pkgs;
```

Comment:
It is not allowed to use pkgs like this in ``pkgs/``.

Reply:
What is the reasoning behind this? What should I use instead?

Reply:
@Kmeakin as you've probably seen in other derivations and in this one (before you changed everything to `pkgs`), it's standard to write every package/builder you need inside the arglist, then call them directly instead of using `pkgs.<package>`.